src/Components/common/contentEntitlement/index.js

```diff
@@ -0,0 +1,51 @@
+import i18n from "../../../i18n";
+import { usePelcro } from "../../../hooks/usePelcro";
+import { notify } from "../../../SubComponents/Notification";
+
+export const init = () => {
+  const { whenSiteReady } = usePelcro.getStore();
+
+  whenSiteReady(() => {
+    const entitlementsProtectedElements = document.querySelectorAll(
+      "[data-entitlement]"
+    );
+
+    entitlementsProtectedElements.forEach((elem) => {
+      const shouldBlurContent = !window.Pelcro.user.isEntitled(
```

Comment:
What's that method? is this new? 
We already have a method for that in the sdk `window.Pelcro.user.isEntitledTo(string)`

Reply:
fixed!